scripts/find_outliers.py

```diff
@@ -22,6 +24,19 @@ def find_outliers(data_directory):
     None
     """
     # Your code here
+
+    ## MLN:  10-5-16
+    # Identify image files in data_directory using names listed in
+    # 'hash_list.txt' (executed similarly to validate_data.py)
+    hashlist = str(data_directory + '/' + 'hash_list.txt')
+    for line in open(hashlist, 'rt'):
+        i = line.split()
+        imgfile = i[1]
+        # Load the image file
+        img = nib.load(imgfile, mmap=False)
+        # Retrieve data from image array
+        data = img.get_data()
+
```

Comment:
Awesome!! 

From here, do we then want to save the data arrays external of the for loop? Because right now each time the data array would be written over with each iteration. 

Maybe we could loop through each time saying a variable called data1, data2, data3 etc. Although we still need to know which data matrix was paired with which .nii file (so a dictionary could help?) 

I'm just kind of thinking outloud, I can accept this for now and then add some more later tonight if that works!